phira/locales/en-US/common.ftl

```diff
@@ -14,19 +14,19 @@ chart-ranked = Ranked
 chart-special = Special
 chart-unstable = Unstable
 
-list-empty = Nothing here
+list-empty = Nothing here.
 
-tos-and-policy = Terms of Service and Privacy Policy
-tos-and-policy-desc = You must read and agree to our Terms of Service and Privacy Policy before using Phira online services provided by TeamFlos.
+tos-and-policy = ToS & Privacy Policy
+tos-and-policy-desc = You must read and accept the Terms of Service and Privacy Policy before using Phira's online services.
 tos-deny = Deny
 tos-accept = Accept
-tos-prev-page = Prev Page
-tos-next-page = Next Page
-loading_tos_policy = Loading Terms of Service and Privacy Policy
-fetch-tos-policy-failed = Failed to fetch Terms of Service and Privacy Policy
-warn-deny-tos-policy = Accept to use online service
+tos-prev-page = Back
+tos-next-page = Next
+loading_tos_policy = Loading the ToS and Privacy Policy...
```

Comment:
Your ToS may be based on whether it will exceed the frame, but this message does not. 
If it references from another standard, please let devs know.

Reply:
what do you mean by "exceed the frame"?  is the text length is too long?

Reply:
Just guess your standard is based on text length. But if that is expected, I'll support you.